spec/index.bs

```diff
@@ -7951,6 +7979,56 @@ When we add HDR canvas output (pixel values > 1), some clarification may be need
 upstream changes to canvas for color and HDR (e.g. to make sure we choose between extended-srgb
 and clamped-srgb: they're the same for SDR, but different for HDR).
 
+### GPUSwapChain sizing ### {#swapchain-sizing}
+
+A {{GPUSwapChain}}'s {{GPUSwapChainDescriptor/size}} is immutable, set by the {{GPUSwapChainDescriptor}}
+passed to {{GPUCanvasContext/configureSwapChain()}}. If a {{GPUSwapChainDescriptor/size}} is not
+specified then the width and height attributes of the {{GPUCanvasContext}}.{{GPUCanvasContext/[[canvas]]}}
+at the time {{GPUCanvasContext/configureSwapChain()}} is called will be used. If the {{GPUSwapChain}}'s
+size does not match the dimensions of the canvas the textures produced by the {{GPUSwapChain}}
+will be scaled to fit the canvas element.
+
+<div class="note">
+    Note: Unlike `'webgl'` or `'2d'` contexts, `width` and `height` attributes of canvases with a
+    `'gpupresent'` context only affect:
+    - Default layout size, if not overridden by CSS.
+    - Default {{GPUSwapChainDescriptor/size}} when calling {{GPUCanvasContext/configureSwapChain()}},
+        if not overridden.
+</div>
+
+In order to match the dimensions of the canvas after it is resized, the {{GPUSwapChain}} must be
+reconfigured by calling {{GPUCanvasContext/configureSwapChain()}} again with the new dimensions.
```

Comment:
nit: should probably clarify that you do not _have_ to "match the dimensions of the canvas"
```suggestion
If it is desired to match the dimensions of the canvas after it is resized, the {{GPUSwapChain}} must be
reconfigured by calling {{GPUCanvasContext/configureSwapChain()}} again with the new dimensions.
```